docs/auth.md

```diff
@@ -0,0 +1,112 @@
+# Task description
+
+* Instead of using one global PostgreSQL connection pool, separate pools should be shared by all client connections with the same username and password.
+
+* FerretDB should handle the client's authentication commands and use provided credentials to authenticate in PostgreSQL.
+
+* Implement createUser, dropUser, etc. commands.
```

Comment:
Would you please list all commands here?
For each command, let's define what parameter will be implemented and what is not.